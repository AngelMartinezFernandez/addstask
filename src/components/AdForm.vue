<template>

</template>

<script>
export default {
  name: "AdForm"
}
</script>

<style scoped>

</style>