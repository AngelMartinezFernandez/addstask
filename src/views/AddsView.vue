<template>
  <div class="home">
    <AdList/>
    <AdForm/>
  </div>
</template>

<script>
import AdForm from '@/components/AdForm.vue';
import AdList from '@/components/AdList.vue';
export default {
  name: 'AddsView',
  components: {
    AdForm, AdList
  }
}
</script>
