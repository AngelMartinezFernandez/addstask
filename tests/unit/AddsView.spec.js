import {shallowMount} from '@vue/test-utils'
import AddsView from '@/views/AddsView'
import AdForm from '@/components/AdForm'
import AdList from '@/components/AdList'

describe('AddsView', () =>{
    it('it renders the component', () => {
        const wrapper = shallowMount(AddsView)
        expect(wrapper.html()).toMatchSnapshot()
    })

    it('it renders right components', () =>{
        const wrapper = shallowMount(AddsView)
        const form = wrapper.findComponent(AdForm)
        const ad = wrapper.findComponent(AdList)

        expect(form.exists()).toBe(true)
        expect(ad.exists()).toBe(true)
    })
})