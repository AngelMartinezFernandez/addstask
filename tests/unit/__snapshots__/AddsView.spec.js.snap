// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddsView it renders the component 1`] = `
<div class="home">
  <ad-list-stub></ad-list-stub>
  <ad-form-stub></ad-form-stub>
</div>
`;
